pages/index.tsx

```diff
@@ -1,19 +1,39 @@
+import { WelcomeToSDC } from "@components/WelcomeToSDC";
 import { useSession, signIn, signOut } from "next-auth/client";
+import { ProgramList } from "../src/components/ProgramList";
+import { Box, Flex, Divider, Spacer, Heading } from "@chakra-ui/react";
 
 export default function Component() {
-    const [session, loading] = useSession();
-    if (session) {
-        return (
-            <>
-                Signed in as {session.user.email} <br />
-                <button onClick={() => signOut()}>Sign out</button>
-            </>
-        );
-    }
+    // const [session, loading] = useSession();
+    // if (session) {
+    //     return (
+    //         <>
+    //             Signed in as {session.user.email} <br />
+    //             <button onClick={() => signOut()}>Sign out</button>
+    //         </>
+    //     );
+    // }
     return (
         <>
-            Not signed in <br />
-            <button onClick={() => signIn()}>Sign in</button>
+            <Flex direction="column" margin="10% 15%">
+                <Box>
+                    <WelcomeToSDC />
+                </Box>
+                <Spacer />
+
+                <Divider
+                    orientation="horizontal"
+                    marginTop="5%"
+                    marginBottom="5%"
```

Comment:
should we be specifying these as percentages? the other approach would be to use Chakra's built-in functionality to make the pages responsive - see [here](https://chakra-ui.com/docs/features/responsive-styles)